app/src/main/java/com/esri/arcgisruntime/opensourceapps/datacollection/views/fragments/PopupAttributesListFragment.kt

```diff
@@ -20,18 +20,25 @@ import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.appcompat.app.AlertDialog
 import androidx.databinding.DataBindingUtil
 import androidx.fragment.app.Fragment
 import androidx.fragment.app.activityViewModels
+import androidx.lifecycle.Observer
 import com.esri.arcgisruntime.opensourceapps.datacollection.R
 import com.esri.arcgisruntime.opensourceapps.datacollection.databinding.FragmentPopupAttributeListBinding
+import com.esri.arcgisruntime.opensourceapps.datacollection.util.observeEvent
 import com.esri.arcgisruntime.opensourceapps.datacollection.viewmodels.IdentifyResultViewModel
+import com.esri.arcgisruntime.opensourceapps.datacollection.viewmodels.PopupViewModel
+import kotlinx.android.synthetic.main.fragment_popup_attribute_list.*
 
 /**
  * Responsible for displaying PopupAttribute list.
  */
 class PopupAttributesListFragment : Fragment() {
```

Comment:
I wonder if we should rename this Fragment, since we don't have a corresponding "PopupAttributesListViewModel". Would it make more sense to rename this to "IdentifyResultFragment"?

Reply:
We already have `IdentifyResultFragment` . `PopupAttributesListFragment` mostly uses the PopupViewModel now . Before that `PopupAttributesListFragment`and `IdentifyResultFragment` shared the IdentifyResultViewModel as they both work off of the same Popup.

Reply:
updated